features.json5

```diff
@@ -0,0 +1,68 @@
+{
+    "features": [
```

Comment:
just to confirm, we're still sticking with features being an array instead of an object with their keys as the feature name right?

Reply:
I vacillated on this, sent you feedback and then deleted it. You're right, an array is better.

When you're joining the features, it is easier to have them as an object. But later when we're passing features around, it is easier to have the feature as *one object* with a name. Otherwise we will be passing `[name, featureData]` tuples everywhere always converting `Object.entries` to a two-pop `{name: ..., data: ...}` with a lot of useless drilling.

So the array is better. We're just not feeling the benefits yet because we're not really doing much joining (although my PR over here does one join for merging... sourcegraph/cody#3956 )